docs/1.0. Certificate Provisioning.md

```diff
@@ -221,126 +225,128 @@ If an EST Client implements the AMWA IS-04 Node API, then the EST Client SHOULD
 
-If the EST Server returns a HTTP 200 response the certificate request was successful and the response from the EST Server SHOULD contain a valid TLS certificate.
+If the EST Server returns an HTTP 200 response, the certificate request was successful and the response from the EST Server SHOULD (?!) contain a valid TLS certificate.
 
-Before using the returned TLS certificate the EST Client SHOULD validate the TLS certificate, validating for example the; NotBefore, Not After, Common Name, Subject Alt Name and chain of trust. The validity of the TLS certificate could be checked by installing the certificate and performing a request to the API. If the TLS certificate is successfully validated, the TLS Certificate and its chain of trust SHOULD be used for all further requests to the NMOS APIs. The EST Client MUST remove any previously issued TLS Certificates and keys. If the EST Client fails to validate the TLS certificate the certificate SHOULD not be used and the [Certificate Request Error Response](#certificate-request-error-response) workflow followed.
+Before using the returned TLS certificate the EST Client SHOULD validate the TLS certificate, validating for example the NotBefore, Not After, Common Name, Subject Alternative Name and chain of trust. The validity of the TLS certificate could be checked by installing the certificate and performing a request to the API. If the TLS certificate is successfully validated, the TLS Certificate and its chain of trust SHOULD be used for all further requests to NMOS APIs. The EST Client MUST remove any previously issued TLS Certificates and keys. If the EST Client fails to validate the TLS certificate, the certificate SHOULD not be used and the [Certificate Request Error Response](#certificate-request-error-response) workflow followed.
 
-If the EST Server returns a HTTP 202 or 503, the request was accepted, but the certificate has not been processed yet. The response SHOULD include a `Retry-After` header and the EST Client MUST not attempt re-sending the request before the defined time has expired. If the TLS session is renegotiated and the CSR includes identity linking information, the CSR will need to be re-generated. The EST Client SHOULD attempt resending the request an appropriate number times before aborting and the [Certificate Request Error Response](#certificate-request-error-response) workflow followed.
+If the EST Server returns an HTTP 202 or 503, the request was accepted, but the certificate has not been processed yet. The response SHOULD include a `Retry-After` header and the EST Client MUST not attempt resending the request before the defined time has expired. If the TLS session is renegotiated and the CSR includes identity linking information, the CSR will need to be regenerated. The EST Client SHOULD attempt resending the request an appropriate number times before aborting and the [Certificate Request Error Response](#certificate-request-error-response) workflow followed.
 
```

Comment:
Suggest that this passage from the "Notes for Implementers" section is either:

- better placed here
- better placed in the implementation guide
- better deleted
```suggestion

Some Base64 decoding libraries expect a line length of 64 characters, although this is not required by [RFC 4648 - Section 3.1](https://tools.ietf.org/html/rfc4648#section-3.1) and some EST Server implementations do not limit the line length of encoded data.
- EST Clients MUST be able to decode Base64 data that is not limited to 64 characters per line.
```